src/main/java/pl/put/poznan/transformer/logic/text/abbreviations/Inverse.java

```diff
@@ -0,0 +1,53 @@
+package pl.put.poznan.transformer.logic.text.abbreviations;
+
+
+import pl.put.poznan.transformer.logic.text.Text;
+import pl.put.poznan.transformer.logic.text.TextTransformer;
+import java.lang.*;
```

Comment:
Nie chcę krzyczeć, ale wildcard importy są złe (w intellij jak klikniesz w _lampkę_ to masz opcje "zamień na pojedyncze importy")